tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html

```diff
@@ -63,11 +63,13 @@
         selected$="[[_expanded]]"
         icon="fullscreen"
         on-tap="_toggleExpanded"
+        title="Make the image fill the whole width. Note that this size may differ from actual."
       ></paper-icon-button>
       <paper-icon-button
+        selected$="[[_actualSizeShown]]"
         icon="aspect-ratio"
-        on-tap="_showActualSize"
-        title="Show the image at its true pixel size"
+        on-tap="_toggleActualSizeShown"
+        title="Show the image at its true pixel size."
```

Comment:
There shouldn't be a period here; tooltips don't have punctuation. (e.g., "Click to go back, hold to see history").

Also, this tooltip is too verbose; suggest "Show actual size".